.scripts/supervisor_mock.py

```
@@ -0,0 +1,274 @@
+                port = 80
+        except ValueError:
+            print(f"Warning: Invalid port '{sys.argv[2]}', using 80")
+            port = 80
+    else:
+        port = 80
+
+    # Bind address: CLI arg > env var > safe default
+    bind_address = (
+        sys.argv[3]
+        if len(sys.argv) > 3
+        else os.environ.get("MOCK_BIND_ADDRESS", "0.0.0.0")
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)